src/services/room-service.spec.js

```diff
@@ -117,4 +121,37 @@ describe('room-service', () => {
     });
 
   });
+
+  describe('isRoomMemberOrOwner', () => {
+    const roomId = uniqueId.create();
+    beforeEach(async () => {
+      await roomStore.save({
+        _id: roomId,
+        name: 'my room',
+        access: ROOM_ACCESS_LEVEL.private,
+        owner: myUser._id,
+        members: [
+          {
+            userId: otherUser._id,
+            joinedOn: new Date().toISOString()
```

Comment:
No need for `.toISOString()`, we always save real dates

Reply:
Oh, okay. fixed.